### ISSUE TITLE
Error creating bean with name 'proxyStarter'

### ISSUE THREAD

**Issue by MysticRyuujin** [emeraldpay/dshackle#8]
I'm running this on Ubuntu 18.04

`sudo docker run -p 2449:2449 -p 8545:8545 -v /etc/dshackle:/etc/dshackle -e "INFURA_USER=$INFURA_USER" emeraldpay/dshackle`

```
Emerald Dshackle - Smart Load Balancer for Blockchain API
   https://github.com/emeraldpay/dshackle
   v0.6.0 built from e3b512ca80 on 2020-04-10T01:30:41 UTC
...
16:14:22.107 [main                ] WARN                  TlsSetup | Using insecure transport for proxy
16:14:22.150 [main                ] WARN  ConfigApplicationContext | Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'proxyStarter': Invocation of init method failed; nested exception is java.lang.NoSuchMethodError: io.netty.channel.SingleThreadEventLoop.<init>(Lio/netty/channel/EventLoopGroup;Ljava/util/concurrent/Executor;ZLjava/util/Queue;Ljava/util/Queue;Lio/netty/util/concurrent/RejectedExecutionHandler;)V
16:14:22.150 [main                ] INFO                GrpcServer | Shutting down GRPC Server...
16:14:22.155 [main                ] INFO                GrpcServer | GRPC Server shot down
16:14:22.165 [main                ] ERROR gFailureAnalysisReporter |

***************************
APPLICATION FAILED TO START
***************************

Description:

An attempt was made to call a method that does not exist. The attempt was made from the following location:

    io.netty.channel.epoll.EpollEventLoop.<init>(EpollEventLoop.java:90)

The following method did not exist:

    io.netty.channel.SingleThreadEventLoop.<init>(Lio/netty/channel/EventLoopGroup;Ljava/util/concurrent/Executor;ZLjava/util/Queue;Ljava/util/Queue;Lio/netty/util/concurrent/RejectedExecutionHandler;)V

The method's class, io.netty.channel.SingleThreadEventLoop, is available from the following locations:

    jar:file:/app/libs/netty-transport-4.1.34.Final.jar!/io/netty/channel/SingleThreadEventLoop.class
    jar:file:/app/libs/netty-all-4.1.48.Final.jar!/io/netty/channel/SingleThreadEventLoop.class

It was loaded from the following location:

    file:/app/libs/netty-transport-4.1.34.Final.jar


Action:

Correct the classpath of your application so that it contains a single, compatible version of io.netty.channel.SingleThreadEventLoop
```

**Comment by splix** [comment]
Oh, it seems that the Docker registry contains some broken build. I'll update it with a fixed version

**Comment by MysticRyuujin** [comment]
Thanks, let me know and I'll try again. Appreciate it. I'm excited to try this out.

**Comment by splix** [comment]
@MysticRyuujin please try again, the issues is fixed in just released v0.6.1. The same docker command should work

**Comment by bessalahAmar** [comment]
@splix hello , i have the same error in my project can you please share with us how you fixed it ?

**Comment by MysticRyuujin** [comment]
When they updated the docker the issue was fixed for me...

I don't have this issue with the latest docker image. I have other issues, but not this one 😄

**Comment by splix** [comment]
@bessalahAmar if you're asking how I fixed it in the code, it was because of duplicate libs for Netty. I had `netty-all`, and individual modules as transients. They were conflicting because of different versions. See the fix 25a872fc9c99

**Comment by bessalahAmar** [comment]
@MysticRyuujin thank you for your answer @splix indeed i'm asking for the code fix , in my case there is no obvious dependecy conflict maybe i have indirect dependecy confilict ,also in my dev environnement i can run the application and it's working but when i ship it inside a docker container i get this error.
